features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/App.jsx

```diff
@@ -73,6 +98,21 @@ class Publisher extends React.Component {
         this.setState({ user });
     }
 
+    /**
+     * Load locale file.
+     *
+     * @param {string} locale Locale name
+     */
+    loadLocale(locale = 'en') {
+        fetch(`${Utils.CONST.CONTEXT_PATH}/public/app/locales/${locale}.json`)
+            .then(resp => resp.json())
+            .then((data) => {
+                // eslint-disable-next-line global-require, import/no-dynamic-require
```

Comment:
please remove commented line

Reply:
Comment is required to skip the eslint issue for no-dynamic-require

Reply:
+1  Since it is a configuration comment.